frontend/src/utils/index.js

```diff
@@ -356,7 +356,7 @@ export function isSelfTerminationWarning (expirationTimestamp) {
 }
 
 export function isValidTerminationDate (expirationTimestamp) {
-  return expirationTimestamp && new Date(expirationTimestamp) > new Date()
+  return !!expirationTimestamp && new Date(expirationTimestamp) > new Date()
```

Comment:
the !! operator is not necessary in this case

Reply:
This is not true
```
undefined && true ->returns undefined
!!undefined && true ->returns false
```